spec/views/event/application_market/_participation.html.haml_spec.rb

```diff
@@ -0,0 +1,15 @@
+#  Copyright (c) 2025, Hitobito AG. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+require "spec_helper"
+
+describe "event/application_market/_participation.html.haml" do
+  let(:event) { EventDecorator.decorate(Fabricate(:course, groups: [groups(:top_layer)])) }
+  let(:participation) { Fabricate(:event_participation, event: event) }
+
+  it "participation without application isn't causing an internal server error" do
```

Comment:
I would call the spec something like "displays participation without application data", instead of saying we don't want an error here